charts/virtual-kubelet-for-aks/templates/deployment.yaml

```diff
@@ -56,5 +56,6 @@ spec:
         hostPath:
           path: /etc/kubernetes/azure.json
           type: File
+      serviceAccountName: {{ if .Values.rbac.install }} "{{ template "fullname" . }}-service-account" {{ end }}
```

Comment:
How about:
```
{{ if .Values.rbac.install }}
      serviceAccountName: {{ template "fullname" . }}
{{ end }}
```

Reply:
I just copied these over as is from the main virtual-kubelet chart. Do you want the following cleaned up for that as well?

Reply:
Yup, appreciate it if you can make them consistent.

Reply:
Another thought: since we have the open PR #258, let's not touch the `virtual-kubelet` chart.

Reply:
👍